cdc/http_router.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdc
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/http/pprof"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap/ticdc/cdc/capture"
+	swaggerFiles "github.com/swaggo/files"
+	ginSwagger "github.com/swaggo/gin-swagger"
+	// use for OpenAPI online docs
+	_ "github.com/pingcap/ticdc/docs"
+)
+
+// NewRouter create a router for OpenAPI
+func NewRouter(capture2 *capture.Capture) (*gin.Engine, error) {
+	router := gin.New()
+
+	// discard gin log output
+	gin.SetMode(gin.ReleaseMode)
+	gin.DefaultWriter = ioutil.Discard
+
+	// request will timeout after 10 second
+	router.Use(timeoutMiddleware(time.Second * 10))
+
+	captureHandler := capture.NewHTTPHandler(capture2)
+
+	// OpenAPI online docs
+	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
+
+	// common API
+	router.GET("/api/v1/status", captureHandler.ServerStatus)
+	router.GET("/api/v1/health", captureHandler.Health)
+	router.GET("api/v1/debug", captureHandler.Debug)
+
+	// changefeed API
+	changefeedGroup := router.Group("/api/v1/changefeeds")
+	{
```

Comment:
unnecessary brackets.

```suggestion
```

Reply:
it is necessary.

Reply:
Could you add some comments? In case we remove these brackets accidently.